packages/core/ios/LottieReactNative/ContainerView.swift

```diff
@@ -210,9 +210,11 @@ class ContainerView: RCTView {
                 onFinish(["isCancelled": !animationFinished])
             }
             self.delegate?.onAnimationFinish(isCancelled: !animationFinished);
+            if (animationFinished) {
+                self.animationView?.currentProgress = 1;
```

Comment:
background behavior `.pauseAndRestore` works fine until the animation ends. If the animation has ended it goes back to the first frame. By setting the progress to the last frame once it finishes, it forces the animation to stay on the last frame.